elements/pfe-tooltip/pfe-tooltip.scss

```diff
@@ -0,0 +1,130 @@
+  --pf-c-tooltip__arrow--m-right--Rotate: 45deg;
+  --pf-c-tooltip__arrow--m-bottom--TranslateX: -50%;
+  --pf-c-tooltip__arrow--m-bottom--TranslateY: -50%;
+  --pf-c-tooltip__arrow--m-bottom--Rotate: 45deg;
+  --pf-c-tooltip__arrow--m-left--TranslateX: 50%;
+  --pf-c-tooltip__arrow--m-left--TranslateY: -50%;
+  --pf-c-tooltip__arrow--m-left--Rotate: 45deg;
+  color: var(--pf-c-tooltip__content--Color);
+  background-color: var(--pf-c-toolip__content--BackgroundColor);
+  line-height: var(--pf-c-tooltip--line-height, 1.5);
+  opacity: 1; 
+  transition: opacity 300ms cubic-bezier(0.54, 1.5, 0.38, 1.11) 0s;
+  position: relative;
+  max-width: var(--pf-c-tooltip--MaxWidth);
+  box-shadow: var(--pf-c-tooltip--BoxShadow);
+  z-index: 999;
+}
+
+[id^=pfe-tooltip][aria-hidden="true"] {
+  opacity: 0; 
+  transition: opacity 300ms cubic-bezier(0.54, 1.5, 0.38, 1.11) 0s;
+}
+
+/** 
+  NOTE: If we replace the popper library the styles related to anything data-popper-* will need to be updated.  
+        These are popper specific data attributes.
+**/
+
+[data-popper-placement='top'] .arrow,
+[data-popper-placement='top-end'] .arrow,
+[data-popper-placement='top-start'] .arrow {
+  bottom: 0;
+  left: 50%;
+  transform: translateX(var(--pf-c-tooltip__arrow--m-top--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-top--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-top--Rotate));
+}
+
+[data-popper-placement='bottom'] .arrow,
+[data-popper-placement='bottom-left'] .arrow,
+[data-popper-placement='bottom-right'] .arrow {
+  top: 0;
+  left: 50%;
+  transform: translateX(var(--pf-c-tooltip__arrow--m-bottom--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-bottom--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-bottom--Rotate));
+}
+
+[data-popper-placement='left'] .arrow,
+[data-popper-placement='left-top'] .arrow,
+[data-popper-placement='left-bottom'] .arrow {
+  top: 50%;
+  right: 0;
+  transform: translateX(var(--pf-c-tooltip__arrow--m-left--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-left--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-left--Rotate));
+}
+
+[data-popper-placement='right'] .arrow,
+[data-popper-placement='right-top'] .arrow,
+[data-popper-placement='right-bottom'] .arrow {
+  top: 50%;
+  left: 0;
+  transform: translateX(var(--pf-c-tooltip__arrow--m-right--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-right--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-right--Rotate));
+}
+
+[data-popper-placement='left-top'] .arrow,
+[data-popper-placement='right-top'] .arrow {
+  top: var(--pf-c-tooltip__arrow--Height);
+}
+
+[data-popper-placement='left-bottom'] .arrow,
+[data-popper-placement='right-right'] .arrow {
+  top: auto;
+  bottom: 0;
+}
+
+[data-popper-placement='top-start'] .arrow,
+[data-popper-placement='bottom-left'] .arrow {
+  left: var(--pf-c-tooltip__arrow--Width);
+}
+
+[data-popper-placement='top-end'] .arrow,
+[data-popper-placement='bottom-right'] .arrow {
+  right: 0;
+  left: auto;
```

Comment:
Fix placement attributes

https://popper.js.org/docs/v1/#popperplacements--codeenumcode

> Valid placements are:
> 
> auto
> top
> right
> bottom
> left
>
> Each placement can have a variation from this list:
> 
> -start
> -end

```Suggestion
[data-popper-placement='top'] .arrow,
[data-popper-placement='top-end'] .arrow,
[data-popper-placement='top-start'] .arrow {
  bottom: 0;
  left: 50%;
  transform: translateX(var(--pf-c-tooltip__arrow--m-top--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-top--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-top--Rotate));
}

[data-popper-placement='bottom'] .arrow,
[data-popper-placement='bottom-start'] .arrow,
[data-popper-placement='bottom-end'] .arrow {
  top: 0;
  left: 50%;
  transform: translateX(var(--pf-c-tooltip__arrow--m-bottom--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-bottom--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-bottom--Rotate));
}

[data-popper-placement='left'] .arrow,
[data-popper-placement='left-start'] .arrow,
[data-popper-placement='left-end'] .arrow {
  top: 50%;
  right: 0;
  transform: translateX(var(--pf-c-tooltip__arrow--m-left--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-left--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-left--Rotate));
}

[data-popper-placement='right'] .arrow,
[data-popper-placement='right-start'] .arrow,
[data-popper-placement='right-end'] .arrow {
  top: 50%;
  left: 0;
  transform: translateX(var(--pf-c-tooltip__arrow--m-right--TranslateX)) translateY(var(--pf-c-tooltip__arrow--m-right--TranslateY)) rotate(var(--pf-c-tooltip__arrow--m-right--Rotate));
}

[data-popper-placement='left-start'] .arrow,
[data-popper-placement='right-start'] .arrow {
  top: var(--pf-c-tooltip__arrow--Height);
}

[data-popper-placement='left-end'] .arrow,
[data-popper-placement='right-end'] .arrow {
  top: auto;
  bottom: 0;
}

[data-popper-placement='top-start'] .arrow,
[data-popper-placement='bottom-start'] .arrow {
  left: var(--pf-c-tooltip__arrow--Width);
}

[data-popper-placement='top-end'] .arrow,
[data-popper-placement='bottom-end'] .arrow {
  right: 0;
  left: auto;
```

Reply:
With the placement changes noted previously we can probably go ahead and remove all of these cases since currently we're only allowing four placement options (top, left, right, bottom).  We can add these classes back in later on as we go in and implement more of the logical properties and rtl functionality.

Reply:
Don't know how I missed those, possibly left over from testing the logical properties changes.  Updated along with the demos.